src/library/scala/collection/Searching.scala

```diff
@@ -16,14 +16,36 @@ import scala.language.implicitConversions
 import scala.collection.generic.IsSeq
 
 object Searching {
+
+  /** The result of performing a search on a sequence
```

Comment:
```suggestion
  /** The result of performing a search on a sorted sequence
```

Maybe we can remind that the underlying sequence has to be sorted first?

Reply:
Yes, I will also remind in the code samples